### PR TITLE
Add more examples for st.metric demo app

### PR DESCRIPTION
**Pull request by asaini**

### CODE REVIEW

**Comment by jrieke** [comment]
@asaini I think you forgot to commit the `iris.csv` file. Also, could you wrap it in a cached function? Should also work like this because the file is very small, but let's rather make sure there's no chance of crashing in case many people visit the demo app...

**Comment by jrieke** [comment]
Or you could also load it directly from sklearn (but not sure if it has the same format): https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_iris.html

File: 0.87/demos/st_metric.py [hunk 1]
```diff
@@ -10,22 +12,75 @@ def show():
...
+        Display KPIs or important metrics in big bold font in your app
+        ---
         """
     )
 
-    st.metric("Streamlit version", 0.87, "0.01")
+    st.subheader('API')
+    st.write("Here's an example which shows the API and `st.metric` in action")
+    with st.echo():
+        st.metric(label="Streamlit version", value=0.87, delta="0.01")
```
Comment (jrieke) [inline 0.87/demos/st_metric.py]:
```suggestion
        st.metric(label="Streamlit version", value=0.87, delta=0.01)
```

(I think I already had string for delta here in the original app, but let's change it to float to make it more consistent for the user.)

File: 0.87/demos/st_metric.py [hunk 2]
```diff
@@ -10,22 +12,75 @@ def show():
...
+    ).configure_legend(
+        orient='bottom'
+    )
+
+    col2.altair_chart(hist, use_container_width=True)
+
+    st.write("---")
+
+    st.write('''Sometimes, an increase in a metric can mean a bad thing. 
+    And a decrease in a metric can be a good. Simply use `delta_color="inverse"` to invert colors for ⬆️ and ⬇️''')
```
Comment (jrieke) [inline 0.87/demos/st_metric.py]:
```suggestion
    And a decrease in a metric can be good. Simply use `delta_color="inverse"` to invert colors for ⬆️ and ⬇️''')
```

File: 0.87/demos/st_metric.py [hunk 3]
```diff
@@ -10,22 +12,75 @@ def show():
...
+    with st.echo():
+        col1, col2, col3 = st.columns(3)
+        col1.metric("Average Query Latency", "1.99ms", "0.25ms", delta_color='inverse')
+        col2.metric("Median Script Execution Time", "0.55ms", "-0.15", delta_color='inverse')
```
Comment (jrieke) [inline 0.87/demos/st_metric.py]:
```suggestion
        col1.metric("Average Query Latency", "1.99ms", "0.25ms", delta_color="inverse")
        col2.metric("Median Script Execution Time", "0.55ms", "-0.15", delta_color="inverse")
```

Just to be consistent with the other code examples ;)

File: 0.87/demos/st_metric.py (hunk 1 above)
Comment (asaini) [reply 0.87/demos/st_metric.py]: Fixed

File: 0.87/demos/st_metric.py (hunk 2 above)
Comment (asaini) [reply 0.87/demos/st_metric.py]: Fixed

File: 0.87/demos/st_metric.py (hunk 3 above)
Comment (asaini) [reply 0.87/demos/st_metric.py]: Fixed

**Comment by asaini** [comment]
Committed `iris.csv`(we now load with st.cache) and also made some changes to `streamlit_app.py`.

**Comment by asaini** [comment]
Going to keep this open. Will merge once v0.87 is out so I don't need a custom wheel file
